src/graph/graph.ts

```diff
@@ -23,23 +27,34 @@ export class Graph<P extends Point2D> {
     return this._points;
   }
 
+  /**
+   * marks all listed points as deleted from graph
+   * @param pointIds points to delete
+   * @private
+   */
+  deletePoints(pointIds: number[]): void {
+    pointIds.forEach((id) => {
+      if (id < 0 || id > Graph.MAX_ID) {
+        return;
+      }
+      this.points[id].deleted = true;
+    });
+  }
+
   /**
    * Creates a Graph instance from a JSON object.
    * @param {P[]} jsonObject - An array of point objects in JSON format.
    * @param {() => P} newObject - A function to create a new point object.
    * @returns {Graph<P>} - A new Graph instance.
    */
-  static fromJson<P extends Point2D>(
-    jsonObject: P[],
-    newObject: (id) => P,
-  ): Graph<P> {
+  static fromJson<P extends Point2D>(jsonObject: P[], newObject: (id: number) => P): Graph<P> {
     return new Graph<P>(
       jsonObject.map((dict) => {
```

Comment:
I receive the following error message when trying to open annotations.

```
Uncaught TypeError: jsonObject.map is not a function
    at Graph.fromJson (graph.ts:52:18)
    at LoadSaveActions.vue:65:47
    at Array.forEach (<anonymous>)
    at LoadSaveActions.vue:64:32
    at Array.forEach (<anonymous>)
    at reader.onload (LoadSaveActions.vue:54:31)
```

Reply:
We changed the format from

[Point]

to [[Point]]

should we support both?

Reply:
It would be good to be backward compatible. At least for now. In the future we can remove it.